File: totosuki/ABC/B/330.py
N, L, R = map(int, input().split())
A = list(map(int, input().split()))
rslt = []

for a in A:
  if a > R:
    rslt.append(R)
  elif a < L:
    rslt.append(L)
  else:
    rslt.append(a)

print(*rslt)

File: OCH_Tao/ABC/A/330.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const L = parseInt(input[0].split(" ")[1], 10);
const A = input[1].split(" ").map(x => parseInt(x, 10));
console.log(A.filter(x => x >= L).length);